client/src/components/DataFiles/DataFilesSystemSelector/DataFilesSystemSelector.jsx

```diff
@@ -48,6 +48,10 @@ const DataFilesSystemSelector = ({
           section,
         },
       });
+      dispatch({
+        type: 'FETCH_SYSTEM_DEFINITION',
+        payload: system.system,
+      });
```

Comment:
Was this needed both here and in `DataFilesShowPathModal.jsx`? Just curious

Reply:
The idea/logic was every time the user selects a different system in the dropdown, it fetches that system definition (so the definition isn't null or undefined). ShowPathModal's dispatch would just be upon the modal opening, and since the modal is already open--my thought was it wouldn't fetch the new system definition without it also being in SystemSelector.